server/cherry_pick.go

```diff
@@ -41,10 +86,16 @@ func (s *Server) handleCherryPick(ctx context.Context, commenter, body string, p
 	if !pr.Merged.Valid || !pr.Merged.Bool {
 		return nil
 	}
-	cmdOut, err := s.doCherryPick(ctx, strings.TrimSpace(args[1]), nil, pr)
-	if err != nil {
-		msg = fmt.Sprintf("Error trying doing the automated Cherry picking. Please do this manually\n\n```\n%s\n```\n", cmdOut)
-		return err
+
+	select {
```

Comment:
My apologies, we do need the case for closing of the channel. Otherwise it will panic once we close the channel. Sorry I forgot about that case earlier.

Reply:
No worries, I got confused as well.